### PR TITLE
cleanup travis

### PR DESCRIPTION
**Pull request by Salzig**

### CODE REVIEW

**Comment by Salzig** [comment]
@januszm any feedback?

**Comment by januszm** [comment]
@Salzig  I'm swamped right now but next week I wanted to merge my PR which also adds support for Rails 5. I've just checked yours and it's a subset of my changes. I think that it makes sense to remove much more dependencies. I'm also wondering why

And if possible, I'd avoid prefixing commit messages with 'feat: ' unless there's a purpose for this (eg. integration with other tools that analyze git history)

**Comment by Salzig** [comment]
@januszm hehe, 'feat:' is just a maybe bad habit. Any problem to merge this?

**Comment by Salzig** [comment]
at least the test suite would be green again.

**Comment by januszm** [comment]
@Salzig having a 'green' test suite asap is not a reason strong enough to make me rework PR for Rails 5. If you can wait just those few days, then I'd merge the bigger PR. Otherwise I'll need to rebase mine on yours (which is OK if you really need your updates before the end of the week).

**Comment by januszm** [comment]
Also, re: commit messages, please try to follow those guidelines, they help maintaining a clean, readable git history

http://chris.beams.io/posts/git-commit/

**Comment by januszm** [comment]
IMHO taggin commit messages makes sense in closed source, commercial projects where you use some kind of issue tracker, or open source, more organized project for the same reason.

**Comment by Salzig** [comment]
@januszm just something i got used to after having contact with angularjs (https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format). Reworded the messages.

**Comment by Salzig** [comment]
IHO: your pull-request contains more then is necessary.

**Comment by Salzig** [comment]
IMHO: i would love to get this merged ASAP. Getting travis green is the first step. After that we can improve in all kind of directions.

**Comment by januszm** [comment]
@Salzig  Angular JS is an example of the 'more organised' opensource project (sponsored by Google) that I mentioned before. Let's leave it, I'd rather stick to a well adopted standards.

re: rebase - It's about lack of time, it's bad if you have to repeat yourself unnecessarily.

Ok let's merge your changes and I'll add my stuff on top of it later.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: ruby
+before_install:
+  - gem install bundler
```
Comment (januszm) [inline .travis.yml]: I'm wondering why did you add this line? Travis comes with bundler. However this line actually fixes my build errors for Ruby 2.1.5. Funny thing is that Travis is using Bundler 1.7.6 with Ruby 2.1.5 and fails to install the gem (should use local source). But on my dev env it works fine, same setup, Ruby 2.1.5 + bundler 1.7.6.

Comment (Salzig) [inline .travis.yml]: Because the old version causes problems.

Comment (Salzig) [inline .travis.yml]: see: https://github.com/travis-ci/travis-ci/issues/5239

**Pull request merged**